src/dsp_tools/commands/xmlupload/xmlupload.py

```diff
@@ -462,6 +474,8 @@ def _create_resource(
 ) -> tuple[str, str] | tuple[None, None]:
     try:
         return resource_create_client.create_resource(resource, bitstream_information)
+    except PermanentConnectionError as err:
+        raise err
```

Comment:
because it would be caught by line 479 and handled differently. I want this function to raise the `PermanentConnectionError`, the other exception handling gives a message on line 480 I do not want.

Reply:
That's true, I didn't realize that. In this case, I would explain this in a short inline comment, for future reference